vars/dockerBuildArgs.groovy

```diff
@@ -48,27 +55,75 @@ String call(Map config = [:]) {
       ret_str += " --build-arg CACHEBUST=${currentBuild.startTimeInMillis}"
       ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
     }
-
     if (add_repos) {
       if (env.REPOSITORY_URL) {
         ret_str += ' --build-arg REPO_URL=' + env.REPOSITORY_URL
       }
-      if (env.DAOS_STACK_EL_7_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_EL7=' + env.DAOS_STACK_EL_7_LOCAL_REPO
-      }
-      if (env.DAOS_STACK_EL_8_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_EL8=' + env.DAOS_STACK_EL_8_LOCAL_REPO
+      string repo_name = null
```

Comment:
I have not quite dug into `string` vs. `String`, but we have been using the latter.